### ISSUE TITLE
I extended cl-yacc, soliciting review

### ISSUE THREAD

**Issue by alanruttenberg** [jech/cl-yacc#4]
Hi,

I extended cl-yacc to add syntax for repetitions, alternates, and optionals. If you are willing, I'd appreciate you doing a review and seeing whether it might be something to integrate into the distributed cl-yacc. I didn't submit as a pull request as it's not a small change. Most of the work is in a separate file, however. 

https://github.com/alanruttenberg/cl-yacc

**Comment by jech** [comment]
I don't think it's a good idea.  It complicates the syntax quite a lot, and I'm not convinced it solves an actual problem, since in practice it is easy enough to add additinal rules.  If you have an example of a grammar where these features are actually useful, please share it with me.

**Comment by alanruttenberg** [comment]
>If you have an example of a grammar where these features are actually useful, please share it with me.

https://github.com/alanruttenberg/lsw2/blob/owlapiv4/logic/cl-bnf.lisp

Implementation of CLIF parser from Common Logic ISO spec.

**Comment by jech** [comment]
Right.  Perhaps I'm in the minority here, but I think that this can easily be achieved using extra rules.  So I'm not going to add support to CL-Yacc.

I'm afraid you'll have to keep maintaining your own fork, sorry for that.
